doc/source/rllib/package_ref/replay_buffers.rst

```diff
@@ -0,0 +1,132 @@
+.. _replay-buffer-api-reference-docs:
+
+Replay Buffer API
```

Comment:
Can we have a quick explain-it-to-a-5yo intro to replay buffers and why users should care here?